docs/src/ice_nucleation_plots/KnopfAlpert2013_fig1.jl

```diff
@@ -0,0 +1,167 @@
+import Plots
+
+import CloudMicrophysics
+import CLIMAParameters
+import Thermodynamics
+
+const PL = Plots
+const IN = CloudMicrophysics.HetIceNucleation
+const CMP = CloudMicrophysics.Parameters
+const CT = CloudMicrophysics.CommonTypes
+const CO = CloudMicrophysics.Common
+const CP = CLIMAParameters
+const TD = Thermodynamics
+
+include(joinpath(pkgdir(CloudMicrophysics), "test", "create_parameters.jl"))
+FT = Float64
+toml_dict = CP.create_toml_dict(FT; dict_type = "alias")
+const prs = cloud_microphysics_parameters(toml_dict)
+thermo_params = CMP.thermodynamics_params(prs)
+
+# Initializing
+temp = collect(210.0:2:232.0) # air temperature
+x = 0.1                       # wt% sulphuric acid in droplets
+Delta_a = Vector{Float64}(undef, length(temp))
+J = Vector{Float64}(undef, length(temp))
+
+# Knopf and Alpert 2013 Figure 4A
+# https://doi.org/10.1039/C3FD00035D
+dust_type = CT.KaoliniteType()
+
+it = 1
+for T in temp
+    Delta_a[it] = CO.Delta_a_w(prs, x, T)
+    J[it] = IN.ABIFM_J(prs, dust_type, Delta_a[it])
+    global it += 1
+end
+log10J_converted = @. log10(J * 1e-4)
+
```

Comment:
Feel free to switch off the formatter in the docs plots scripts or data points, where it makes sense. 

You can do it like this

```
#! format: off 

KA13_Delta_a_obs = [0.13641, 0.16205, 0.21538, 0.23897, more numbers, ...]

.. some more long arrays

#! format: on  
```